package.json

```diff
@@ -37,7 +37,7 @@
     "coverage:ci": "node packages/build/bin/run-nyc report --reporter=text-lcov | coveralls",
     "precoverage": "npm test",
     "coverage": "open coverage/index.html",
-    "lint": "npm run prettier:check && npm run tslint",
+    "lint": "npm run prettier:check && npm run tslint && node bin/check-package-locks",
```

Comment:
Have you considered having this check in a different build step than `lint`?

Reply:
Initially, I put the check into `.travis.yml` only. But then I realized that we don't want developers to discover problem with package-locks only after opening a pull request. It's better to find that out while running `npm test`.

Please note that this check is pretty cheap, it takes less than a second on my machine:

```
$ time bin/check-package-locks.js

real	0m0.729s
user	0m0.337s
sys	0m0.191s
```

What are your concerns? Why do you think this should go to a different script? I suppose I can move it to `posttest` if you prefer.

Reply:
I was just wondering why we put it there, sounds good. Thank you for providing the tests too!